src/acp/ess1.ts

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright 2022 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * :::{admonition} Experimental API
+ * :class: important
+ *
+ * The Access Control Policies proposal has not yet been reviewed for inclusion in the Solid spec.
+ * To enable early experimentation, solid-client exposes a low-level API. However, this API can and
+ * will include breaking changes in non-major releases. Additionally, for most applications, a
+ * higher-level API that is planned will be more applicable.
+ *
+ * Thus, the following export is *only* intended for experimentation by early adopters, and is not
+ * recommended for production applications. Because of this, all ACP-related API's are exported on a
+ * single object, which does not facilitate tree-shaking: if you use one ACP-related API, all of
+ * them will be included in your bundle.
+ *
+ * For more information see: [Tutorial: Managing
+ * Access](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/manage-access/)
+ * :::
+ *
+ * This module can be imported as an object from the main package, which results
```

Comment:
Is this to support ess 1.1 and earlier?  If so, probably should mention it?  otherwise, it's just acp_ess_1 and acp_ess_2 and people will have to guess at which. And maybe state this replaces acp_v3 or whatever it used to be (for people using PodSpaces and who won't care about ESS versions directly)?

Reply:
I made an explicit statement about ESS 1.1 and ESS 2.0 support in https://github.com/inrupt/solid-client-js/commit/6c6385d4f7ddd376053d3825011ab124b2ec1a3f.

There is already a deprecation notice in acp_v3 pointing to this module (in "[src/acp/v3.ts](https://github.com/inrupt/solid-client-js/pull/1564/commits/c4d5957d98993aefb45ade380a09aae53fc87175#diff-ddf00bafa60da66f28fd8d0b033aca4913fc95a3abeaed01e22c8ca2b3ac011d)".